katdal/h5datav3.py

```diff
@@ -422,6 +422,10 @@ def register_telstate_sensor(name, obj):
         # Store antenna objects in sensor cache too, for use in virtual sensor calculations
         for ant in ants:
             self.sensor['Antennas/%s/antenna' % (ant.name,)] = CategoricalData([ant], [0, num_dumps])
+        # Extract array reference from first antenna (first 5 fields of description)
+        array_ant_fields = ['array'] + ants[0].description.split(',')[1:5]
+        array_ant = katpoint.Antenna(','.join(array_ant_fields))
```

Comment:
This is getting copy-pasted a lot. I'll approve for now, but I'd really like to see a helper function in katpoint for this.

Reply:
Indeed. I've made a note of this in SR-1619.